### PR TITLE
ActiveJob Limiter Plugin

### PR DESCRIPTION
**Pull request by flexportnes**
💥

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+And then execute:
+
+    $ bundle
+
+Or install it yourself as:
+
+    $ gem install activejob-limiter
+
+## Usage
```
Comment (Gasparila) [inline README.md:21]: Maybe add examples of why this is useful?

File: activejob-limiter.gemspec [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  spec.license       = 'MIT'
+
+  # Specify which files should be added to the gem when it is released.
+  spec.files = Dir['{lib}/**/*', 'README.md']
+  spec.test_files = Dir['spec/**/*']
+
+  spec.require_paths = ['lib']
+
+  spec.add_dependency 'activejob', '~> 5.1.6.1'
+  spec.add_dependency 'activesupport', '~> 5.1.6.1'
```
Comment (Gasparila) [inline activejob-limiter.gemspec]: Can we relax these requirements at all?

File: bin/console [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+# You can add fixtures and/or initialization code here to make experimenting
+# with your gem easier. You can also use a different console, if you like.
+
+# (If you use this, don't forget to add pry to your Gemfile!)
+# require 'pry'
+# Pry.start
+
+require 'irb'
+IRB.start(__FILE__)
```
Comment (Gasparila) [inline bin/console:15]: 🔥 these unneeded scaffold files?

File: spec/active_job/limiter_spec.rb [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+  it 'checks a lock before enqueuing a job' do
+    allow(ActiveJob::Limiter).to receive(:clear_lock_before_perform).and_return(true)
+    expect(ActiveJob::Limiter).to receive(:check_lock_before_enqueue)
+      .with(instance_of(LimitedJob), expiration_time).and_return(true)
+    LimitedJob.perform_later
+    expect(performed_jobs.size).to eq(1)
+  end
+
+  it 'does not enqueue the job if the lock check returns false' do
```
Comment (Gasparila) [inline spec/active_job/limiter_spec.rb]: Can we add a spec to check what happens if enqueueing/performing raise errors?

**Review by Gasparila — CHANGES_REQUESTED** [review]
Looking food, left some minor comments

File: bin/console (hunk 3 above)
Comment (flexportnes) [reply bin/console:15]: They're generated by Bundler when you make a new gem. They do provide functionality so I didn't think it was necessary to delete them.

File: activejob-limiter.gemspec (hunk 2 above)
Comment (flexportnes) [reply activejob-limiter.gemspec]: Yes!

**Review by Gasparila — APPROVED** [review]

**Pull request merged**